app/routes/titiler/algorithms/alerts.py

```diff
@@ -9,31 +9,29 @@
 
 from app.models.enumerators.alerts_confidence import DeforestationAlertConfidence
 
-START_DATE: str = "2014-12-31"  # start of record
-
-ALERT_CONFIDENCE_MAP: OrderedDict = OrderedDict(
-    {
-        DeforestationAlertConfidence.low: {
-            "confidence": 2,
-            "colors": {"red": 237, "green": 164, "blue": 194},
-        },
-        DeforestationAlertConfidence.high: {
-            "confidence": 3,
-            "colors": {"red": 220, "green": 102, "blue": 153},
-        },
-        DeforestationAlertConfidence.highest: {
-            "confidence": 4,
-            "colors": {"red": 201, "green": 42, "blue": 109},
-        },
-    }
-)
-
-
-class IntegratedAlerts(BaseAlgorithm):
-    """Decode Integrated Alerts."""
-
-    title: str = "Integrated Deforestation Alerts"
+
+class Alerts(BaseAlgorithm):
+    """Decode Deforestation Alerts."""
+
+    title: str = "Deforestation Alerts"
     description: str = "Decode and visualize alerts"
+    conf_colors: OrderedDict = OrderedDict(
+        {
+            DeforestationAlertConfidence.low: {
+                "confidence": 2,
+                "colors": (237, 164, 194),
+            },
+            DeforestationAlertConfidence.high: {
+                "confidence": 3,
+                "colors": (220, 102, 153),
+            },
+            DeforestationAlertConfidence.highest: {
+                "confidence": 4,
+                "colors": (201, 42, 109),
+            },
+        }
+    )
+    record_start_date: str = "2014-12-31"
 
     # Parameters
     default_start_date: str = (date.today() - relativedelta(days=180)).strftime(
```

Comment:
This is a nitpick but I thought I'd mention it:
I noticed that `data.today()` is called a couple of times. Is it worth it to just call it once? 🤷

Reply:
Always good to DRY,  let me update it.